cdap-app-fabric/src/main/java/io/cdap/cdap/security/impersonation/DefaultUGIProvider.java

```diff
@@ -50,15 +62,19 @@
   private final LocationFactory locationFactory;
   private final File tempDir;
   private final NamespaceQueryAdmin namespaceQueryAdmin;
+  protected final Store store;
+  private static final String RUNTIME_ARG_KEYTAB = "pipeline.keytab.path";
```

Comment:
Avoid repeating constant definition in different classes. As mentioned above, move definition into `SystemArguments` class and have helper method in that class to help using these arguments.